models/sessions/sessions.yml

```diff
@@ -87,140 +87,263 @@ models:
       - name: mobile__apple_idfa
+        description: '{{ doc("col_apple_idfa") }}'
       - name: mobile__apple_idfv
-      - name: mobile__carrier
+        description: '{{ doc("col_apple_idfv") }}'
       - name: mobile__open_idfa
+        description: '{{ doc("col_open_idfa") }}'
       - name: mobile__network_technology
+        description: '{{ doc("col_network_technology") }}'
       - name: mobile__network_type
+        description: '{{ doc("col_network_type") }}'
       - name: mobile__physical_memory
+        description: '{{ doc("col_physical_memory") }}'
       - name: mobile__system_available_memory
+        description: '{{ doc("col_system_available_memory") }}'
       - name: mobile__app_available_memory
+        description: '{{ doc("col_app_available_memory") }}'
       - name: mobile__battery_level
+        description: '{{ doc("col_battery_level") }}'
       - name: mobile__battery_state
+        description: '{{ doc("col_battery_state") }}'
       - name: mobile__low_power_mode
+        description: '{{ doc("col_low_power_mode") }}'
       - name: mobile__available_storage
+        description: '{{ doc("col_available_storage") }}'
       - name: mobile__total_storage
+        description: '{{ doc("col_total_storage") }}'
       - name: mobile__is_portrait
+        description: '{{ doc("col_is_portrait") }}'
       - name: mobile__resolution
+        description: '{{ doc("col_resolution") }}'
       - name: mobile__scale
+        description: '{{ doc("col_scale") }}'
       - name: mobile__language
+        description: '{{ doc("col_language") }}'
       - name: mobile__app_set_id
+        description: '{{ doc("col_app_set_id") }}'
       - name: mobile__app_set_id_scope
-      - name: first_geo_country
-      - name: first_geo_region_name
-      - name: first_geo_city
+        description: '{{ doc("col_app_set_id_scope") }}'
+        description: '{{ doc("col_geo_country") }}'
       - name: first_geo_country_name
+        description: '{{ doc("col_geo_country_name") }} {{ doc("first_session") }}'
       - name: first_geo_continent
+        description: '{{ doc("col_geo_continent") }} {{ doc("first_session") }}'
+      - name: first_geo_city
+        description: '{{ doc("col_geo_city") }} {{ doc("first_session") }}'
+      - name: first_geo_region_name
+        description: '{{ doc("col_geo_region_name") }} {{ doc("first_session") }}'
       - name: last_geo_country
-      - name: last_geo_region_name
-      - name: last_geo_city
+        description: '{{ doc("col_geo_country") }}'
       - name: last_geo_country_name
+        description: '{{ doc("col_geo_country_name") }}'
       - name: last_geo_continent
+        description: '{{ doc("col_geo_continent") }}'
+      - name: last_geo_city
+        description: '{{ doc("col_geo_city") }}'
+      - name: last_geo_region_name
+        description: '{{ doc("col_geo_region_name") }}'
       - name: geo_zipcode
+        description: '{{ doc("col_geo_zipcode") }}'
```

Comment:
All of these are firsts, too. If you check the sessions this run, anything with f. is first l. is last. The first_ / last_ ones are easy to find regardless, for these it is more important I would think.